File: Leet Code/27. Remove Element/27. Remove Element.java
class Solution {
    public int removeElement(int[] nums, int val) {
        int count = 0;
        for (int i = 0; i<nums.length; i++){
            //System.out.println(i);
            //System.out.println(nums.length-1);
            if(i <= nums.length - 1 && nums[i]!= val){

                nums[count] = nums[i];
                count++;
                //continue;
            }
            else if (i < nums.length - 1 && nums[i]== val)
            {
                continue;
            }
            //nums[count] = nums[i];


        }
        /*for (int i = 0 ; i<nums.length; i++){
            System.out.print(" "+nums[i]);
        }*/
        return count;
    }
}

File: Leet Code/27. Remove Element/27. Remove Element.kt
class Solution {
    fun removeElement(nums: IntArray, val_: Int): Int {
        var count = 0
    val size = nums.size-1
    for (i in 0..nums.size-1){
        if(i<=size && nums[i] != val_){
            nums[count] = nums[i]
            count += 1
        }
        else if(i<size && nums[i]==val_){
            continue
        }
    }
    
    return count
    }
}
